script/bootstrap

```diff
@@ -41,17 +41,11 @@ install_packages_apt()
 {
     # apt update and install dependencies
     sudo apt-get update
-    sudo apt-get install -y build-essential git make autoconf \
-        autoconf-archive automake dbus libtool gcc \
-        g++ gperf flex bison texinfo ncurses-dev \
-        libexpat-dev python sed python-pip gawk \
-        libreadline6-dev libreadline6 libdbus-1-dev \
-        libboost-dev
+    sudo apt-get install automake libtool
```

Comment:
I tried on latest ubuntu docker image.
+ *make* is not installed.
+ *g++* is not installed.

So, I think we need
```sh
sudo apt-get install automake libtool make g++
```